### PR TITLE
remove unsupported test case

### PR DESCRIPTION
**Pull request by BaoshanPang**
r? @alexcrichton

### CODE REVIEW

File: src/libstd/sys/vxworks/process/process_common.rs [hunk 1]
```diff
@@ -404,66 +404,3 @@ impl ExitCode {
     }
 }
```
Comment (JohnTitor) [inline src/libstd/sys/vxworks/process/process_common.rs]: This blank line should be removed.

**Comment by alexcrichton** [comment]
@BaoshanPang this is presumably an unsupported test case on vxworks, right? I think it's supported elsewhere though so could this only be excluded on vxworks?

**Comment by BaoshanPang** [comment]
> @BaoshanPang this is presumably an unsupported test case on vxworks, right? I think it's supported elsewhere though so could this only be excluded on vxworks?

The test case is using 'cat' which is not existed on vxWorks. We maybe want to develop an equal test case in futcher, but for now just remove it.

**Comment by alexcrichton** [comment]
@bors: r+ rollup

Oh oops sorry I forgot this was already in a vxworks-specific module

**Comment by BaoshanPang** [comment]
> @bors: r+ rollup
> 
> Oh oops sorry I forgot this was already in a vxworks-specific module

Thanks for the quick reivew as always.

**Pull request merged**